estuary_updater/handlers/base.py

```diff
@@ -105,9 +123,14 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
             log.error('Failed to get brew build using the identifier {0}'.format(identifier))
             raise
 
+        try:
+            extra_json = json.dumps(build_info['extra'])
```

Comment:
You want to only do this if `extra_json` has a value. Otherwise `json.dumps` will convert this to a string that just says null.